libpod/networking_linux.go

```diff
@@ -63,14 +63,14 @@ func (r *Runtime) configureNetNS(ctr *Container, ctrNS ns.NetNS) (err error) {
 		ctr.state.NetworkStatus = append(ctr.state.NetworkStatus, resultCurrent)
 	}
 
-	for _, r := range ctr.state.NetworkStatus {
-		// We need to temporarily use iptables to allow the container
-		// to resolve DNS until this issue is fixed upstream.
-		// https://github.com/containernetworking/plugins/pull/75
-		for _, ip := range r.IPs {
-			if ip.Address.IP.To4() != nil {
-				iptablesDNS("-I", ip.Address.IP.String())
-			}
+	// Add firewall rules to ensure the container has network access.
+	// Will not be necessary once CNI firewall plugin merges upstream.
+	for _, netStatus := range ctr.state.NetworkStatus {
+		firewallConf := &firewall.FirewallNetConf{
+			PrevResult: netStatus,
+		}
+		if err := r.firewallBackend.Add(firewallConf); err != nil {
+			return errors.Wrapf(err, "error adding firewall rules for container %s", ctr.ID())
```

Comment:
Is there an active PR upstream?  If so, might be nice to add a link,if not, ignore this comment.

Reply:
https://github.com/containernetworking/plugins/pull/75